### PR TITLE
Feature/user main page

### PR DESCRIPTION
**Pull request by azatibraev**
Get favorite books in the main  page

### CODE REVIEW

File: src/main/java/kg/eBook/ebookb5/services/BookService.java [hunk 1]
```diff
@@ -169,11 +175,56 @@ public Page<AdminBooksResponse> findAllBooks(Long genreId,
...
+                bookType,
+                pageable);
     }
 
 
+    public MainPageResponse mainPageResponse() {
+
+        List<Long> favoriteBooks = bookRepository.findAllFavoriteBooks();
+        List<Book> allFavoriteBooks = new ArrayList<>();
+        for (Long favoriteBook : favoriteBooks) {
```
Comment (beksultancs) [inline src/main/java/kg/eBook/ebookb5/services/BookService.java:189]: Myndai kyla alasyzby
srazy ele booktardy alyp chyga turgan
a to sizdiki birinchi idlerin alyp anan kaira birden alyp chykkan ayabai kop ubakytty alat

**Review by beksultancs — CHANGES_REQUESTED** [review]
Need to change something!

File: src/main/java/kg/eBook/ebookb5/services/BookService.java (hunk 1 above)
Comment (azatibraev) [reply src/main/java/kg/eBook/ebookb5/services/BookService.java]: changed

**Comment by azatibraev** [comment]
I hope it's okie dokie

**Pull request merged**